.azure/pipelines/ci.yml

```diff
@@ -527,6 +527,7 @@ stages:
       jobDisplayName: "Test: Windows Server 2016 x64"
       agentOs: Windows
       isTestingJob: true
+      useHostedUbuntu: false
```

Comment:
This seems like a weird place to put this - it's in the `Windows_Test` section?

Reply:
Yup. I moved it down into the Ubuntu test job where we want it.